src/plugins/python/pythonplugin.cpp

```diff
@@ -40,7 +41,8 @@ static void handleError()
 }
 
 PythonPlugin::PythonPlugin()
-    : mScriptDir(QDir::homePath() + "/.tiled")
+    : mSysScriptDir(QCoreApplication::applicationDirPath() + "/scripts")
```

Comment:
Probably would prefer to call this `mAppScriptDir`.

Unfortunately, initializing it like this will only work on Windows. Check out the `LanguageManager` constructor for how we could make it work on macOS and Linux (and of course, install rules will need to be written for this - see `translations.qbs`).